python/dynamic_graph/sot/torque_control/tests/test_torque_offset_estimator.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+"""
+2017, LAAS/CNRS
+@author: Rohan Budhiraja
+"""
+
+
+
+# ______________________________________________________________________________
+# ******************************************************************************
+#
+#1)  The simplest robot task: Just go and reach a point
+#2)  While performing this task:
+# )               find the torque values and feed them to the sensor calibration entity.
+#3)  Confirm that the offsets being calculated are correct.
+# ______________________________________________________________________________
+# ******************************************************************************
+
+# Requires sot-dynamic-pinocchio
+
+#-----------------------------------------------------------------------------
+
+
+#SET THE PATH TO THE URDF AND MESHES
+#Define robotName, urdfpath and initialConfig
+dt = 5e-3
+urdfPath = "/local/rbudhira/git/pyrene/talos-data/robots/talos_reduced.urdf"
```

Comment:
Since several people are starting to use this repository we should stop hard-coding the path to the urdf file in the code. What I suggest you to do is to put the path to the urdf file in an environment variable, and then go read it in the python file. Something like this:
```
ERROR_MSG                   = 'You should set the environment variable TALOS_MODEL_DIR to something like "$DEVEL_DIR/pyrene/talos-data"\n';
urdfDir                  = [os.environ.get('TALOS_MODEL_DIR', ERROR_MSG)];
urdfPath                = urdfDir[0] + "/robots/talos_reduced.urdf";
```